test/AxelarGatewayMultisig.js

```diff
@@ -599,40 +599,58 @@ describe('AxelarGatewayMultisig', () => {
     });
 
     describe('command transferOwnership', () => {
-      it('should owners to transfer ownership', () => {
-        const data = arrayify(
-          defaultAbiCoder.encode(
-            ['uint256', 'uint256', 'bytes32[]', 'string[]', 'bytes[]'],
-            [
-              CHAIN_ID,
-              ROLE_OWNER,
-              [getRandomID()],
-              ['transferOwnership'],
+      it('should owners to transfer ownership', async () => {
+        const makeTransferCommand = (newThreshold) =>
```

Comment:
I would extract this function to the top of the file so that you don't need to define it twice inline

Reply:
Will do that to make things cleaner